nltk/test/unit/test_util.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for nltk.util.
+"""
+
+import unittest
+from nltk.util import everygrams
+
+
+class TestEverygrams(unittest.TestCase):
+
+    def setUp(self):
+        """Form test data for tests."""
+        self.test_data = 'a b c'.split()
+
+    def test_everygrams(self):
+        # Test everygrams without padding.
```

Comment:
Let's move this comment into the function name: `test_everygrams_without_padding`

Reply:
Done.